airbyte-integrations/connectors/source-public-apis/integration_tests/abnormal_state.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Since it doesn't support incremental, the file used to test incremental import can be removed.